frontend/testing/playwright/components/SettingsModal.ts

```diff
@@ -1,10 +1,8 @@
+import type { SettingsModalTab } from 'app-development/types/SettingsModalTab';
 import { BasePage } from '../helpers/BasePage';
 import type { Environment } from '../helpers/StudioEnvironment';
 import type { Page } from '@playwright/test';
 
-export type SettingsModalTab = 'about' | 'accessControl' | 'policy' | 'setup';
```

Comment:
This should not be removed. 
we created this “duplicate” to not be dependent on code from the packages

Reply:
However, it might be changed to this: 
```tsx
SettingsModalTab = 'about' | 'access-control' | 'policy' | 'setup';
```

Reply:
Done. Or with underscore :D